bundle/src/projects/commercial/modules/header-bidding/types.d.ts

```diff
@@ -11,9 +11,13 @@ declare global {
 		| 'mostpop'
 		| 'right'
 		| 'top-above-nav'
+		| `fronts-banner-${number}`
```

Comment:
could we lose the `-` before the number for consistency with `inline`?

Reply:
I did have a think about this, but to me I don't think that `fronts-banner1` is in itself a consistent format without the dash before the number. I don't think that it would be worth maintaining consistency with the `inline` slots. Happy to change if I'm outvoted though.

Reply:
I guess I see both arguments now, I leave it to you 👍